manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -115,3 +115,8 @@ spec:
         apiserver_request_count{group="extensions",version="v1beta1",resource!~"ingresses|",client!~"hyperkube/.*|cluster-policy-controller/.*"}
       labels:
         severity: warning
+  - name: kube-apiserver-max-in-flight
+    interval: 5s
+    rules:
+    - record: cluster:apiserver_current_inflight_requests:sum
+      expr: 'sum(apiserver_current_inflight_requests{namespace=~"openshift-apiserver|default"}) by (requestKind,namespace)'
```

Comment:
This namespace is probably the one of the requests, not of the apiserver container.

Reply:
@sttts I just checked the documentation - I think `namespace` points to the source, it's the namespace of the `Service`, `Endpoints` from which the metrics are scraped. In case of `kube-apiserver` it is the Endpoint `kubernetes` in `default` namespace.

Reply:
`apiserver_current_inflight_requests` does not declare the `namespace` label itself, hence its value should indeed be the namespace location of the apiserver container.

That metric only sets `requestKind` as per https://github.com/kubernetes/kubernetes/blob/703a719557d8208f781c88c16816158cf0c98d22/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L143-L152 and https://github.com/kubernetes/kubernetes/blob/703a719557d8208f781c88c16816158cf0c98d22/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L203-L206